From 7c5b281bf4403de8a02a3793188a4d735efafe9a Mon Sep 17 00:00:00 2001
From: epideme <tony.nilsson@epideme.com>
Date: Sun, 16 Sep 2012 19:57:52 +0300
Subject: [PATCH] [rbp] Allow OMXPlayer to try playing DVD/BD-iso

Now that scaling is fixed and overlays behave better, and license for MPG2 codec is available, allow iso playback with a warning in the debug log.
---
 xbmc/cores/omxplayer/OMXPlayer.cpp |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/xbmc/cores/omxplayer/OMXPlayer.cpp b/xbmc/cores/omxplayer/OMXPlayer.cpp
index a3736df..1fc32b4 100644
--- a/xbmc/cores/omxplayer/OMXPlayer.cpp
+++ b/xbmc/cores/omxplayer/OMXPlayer.cpp
@@ -590,8 +590,8 @@ bool COMXPlayer::OpenInputStream()
   if (m_pInputStream && ( m_pInputStream->IsStreamType(DVDSTREAM_TYPE_DVD)
                        || m_pInputStream->IsStreamType(DVDSTREAM_TYPE_BLURAY) ) )
   {
-    CLog::Log(LOGINFO, "COMXPlayer::OpenInputStream - DVD/BD not supported");
-    return false;
+    CLog::Log(LOGINFO, "COMXPlayer::OpenInputStream - DVD/BD not supported - Will try...");
+    // return false;
   }
 
   // find any available external subtitles for non dvd files
-- 
1.7.10

